mod/subthread.php

```diff
@@ -41,8 +41,9 @@ function subthread_content(&$a) {
 			intval($item['contact-id']),
 			intval($item['uid'])
 		);
-		if(! dbm::is_result($r))
+		if (! dbm::is_result($r)) {
 			return;
+		}
 		if(! $r[0]['self'])
```

Comment:
Standards: Could you please add brackets to this conditional statement? 😊
Standards: Could you please add a space after the `if`? 😊